snet-ui/Noteligible/index.tsx

```diff
@@ -0,0 +1,122 @@
+import React, { useEffect } from "react";
+import Typography from "@mui/material/Typography";
+
+import { Box } from "@mui/system";
+
+import { Stack } from "@mui/material";
+import Button from "@mui/material/Button";
+
+export default function Notqualified() {
+  return (
+    <Box
+      sx={{
+        height: "1150px",
+        width: "1440px",
+      }}
+    >
+      <Box
+        sx={{
+          bgcolor: "bgHighlight.main",
+          height: "932px",
+          width: "1160px",
+          m: 11,
+          pb: 2,
+        }}
+      >
+        <Box
+          sx={{
+            display: "flex",
+            justifyContent: "center",
+
+            pb: 3,
+          }}
+        >
+          <Box sx={{ height: "311px", width: "765px" }}>
+            <Box sx={{ m: 2 }}>
+              <Typography
+                align="center"
+                fontWeight="bold"
+                variant="h5"
+                color="redtext.main"
+                fontFamily="Montserrat"
```

Comment:
don't use fontFamily, fontWeight. Both of them should come directly from the theme. 
You have to just use the variant.